Matrix.ConsoleApp/MatrixExtension.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+
+namespace Matrix.ConsoleApp
+{
+    internal static class MatrixExtension 
+    {
+        public static Matrix GetNewMatrixFromAddingTwoMatrices(this Matrix first, Matrix second)
+        {
+            //adding elements of two arrays
+            int[] array = new int[first.Size];
+            if (first.Size > second.Size)
```

Comment:
Complete the requirement: If the dimensions of the matrix do not match, the smaller matrix is padded with zeros.